package.json

```diff
@@ -18,7 +18,7 @@
     "url": "https://github.com/prismatic-io/translations.git"
   },
   "license": "MIT",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
I'd probably bump to 1.7.0 instead as this isn't a bugfix but adding additional translations.